docs/pages/use-teleport/tsh.mdx

```diff
@@ -764,5 +764,55 @@ $ tsh status
   Extensions:         permit-X11-forwarding
 ```
 
+## Custom aliases and defaults
+
+You can define your own custom commands and set command-specific flag defaults by using the `aliases` section of `tsh` config.
```

Comment:
```suggestion
You can configure `tsh` to define aliases, custom commands and command-specific flag defaults. Using aliases, you can run frequently used `tsh` commands more easily.
```
We haven't introduced the `tsh` config file yet, so I think we should lead with the benefit of this feature.